ucs/database.h

```diff
@@ -42,9 +44,16 @@ class UCSDatabase : public Database {
 	bool VerifyMailKey(std::string CharacterName, int IPAddress, std::string MailKey);
 	bool GetVariable(const char* varname, char* varvalue, uint16 varvalue_len);
 	bool LoadChatChannels();
+	void LoadReservedNamesFromDB();
+	bool IsChatChannelInDB(std::string channelName);
+	bool CheckChannelNameFilter(std::string channel_name);
+	void SaveChatChannel(std::string channelName, std::string channelOwner, std::string channelPassword, uint16 minstatus);
+	void DeleteChatChannel(std::string channelName);
+	int CurrentPlayerChannelCount(const std::string& PlayerName);
+	std::string CurrentPlayerChannels(std::string PlayerName);
 	void GetAccountStatus(Client *c);
-	void SetChannelPassword(std::string ChannelName, std::string Password);
-	void SetChannelOwner(std::string ChannelName, std::string Owner);
+	void SetChannelPassword(const std::string& ChannelName, const std::string& Password);
+	void SetChannelOwner(const std::string& ChannelName, const std::string& owner);
```

Comment:
Should these be `const std::string&` to avoid making a copy? Also `channel_name`, `channel_owner`, `channel_password`, `player_name`, and `password`?